### PR TITLE
Update keyvault.md Java for the GA release

### PR DESCRIPTION
**Pull request by elraikhm**
Please review and let me know if this works

### CODE REVIEW

File: docs-ref-services/keyvault.md [hunk 1]
```diff
@@ -21,32 +21,54 @@ Safeguard and manage cryptographic keys and secrets used by cloud applications a
...
+| Key Vault - Secrets | [azure-security-keyvault-secrets - 4.0.0](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets/4.0.0/jar/) | [ReadMe](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-secrets_4.0.0/sdk/keyvault/azure-security-keyvault-secrets/README.md) | [Samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-secrets_4.0.0/sdk/keyvault/azure-security-keyvault-secrets/src/samples) | [Api Reference](https://azuresdkdocs.blob.core.windows.net/$web/java/azure-security-keyvault-secrets/4.0.0/index.html) | [ChangeLog](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-secrets_4.0.0/sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md) |
 
-> [!div class="nextstepaction"]
```
Comment (kaerm) [inline docs-ref-services/keyvault.md]: I'd keep this and as an underlying link I would point to the QS for secrets: [Get started with Secrets](/azure/key-vault/quick-create-java)

File: docs-ref-services/keyvault.md [hunk 2]
```diff
@@ -21,32 +21,54 @@ Safeguard and manage cryptographic keys and secrets used by cloud applications a
...
+SecretClient secretClient = new SecretClientBuilder()
+		 .vaultUrl(<your-vault-url>)
+		 .credential(new DefaultAzureCredentialBuilder().build())
+		 .buildClient();
+KeyVaultSecret secret = secretClient.getSecret("secret_name");
+
 ```
+| Service | Package | Readme | Samples | API Reference | Changelog |
+| ------- | ------- | ------ | ------- | ------------- | --------- |
+| Key Vault - Keys | [azure-security-keyvault-keys - 4.0.0](https://search.maven.org/artifact/com.azure/azure-security-keyvault-keys/4.0.0/jar/) | [ReadMe](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-keys_4.0.0/sdk/keyvault/azure-security-keyvault-keys/README.md) | [Samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-keys_4.0.0/sdk/keyvault/azure-security-keyvault-keys/src/samples) | [Api Reference](https://azuresdkdocs.blob.core.windows.net/$web/java/azure-security-keyvault-keys/4.0.0/index.html) | [ChangeLog](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-keys_4.0.0/sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md) |
```
Comment (kaerm) [inline docs-ref-services/keyvault.md]: nit: I'd use README

File: docs-ref-services/keyvault.md [hunk 3]
```diff
@@ -21,32 +21,54 @@ Safeguard and manage cryptographic keys and secrets used by cloud applications a
...
+| Key Vault - Secrets | [azure-security-keyvault-secrets - 4.0.0](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets/4.0.0/jar/) | [ReadMe](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-secrets_4.0.0/sdk/keyvault/azure-security-keyvault-secrets/README.md) | [Samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-secrets_4.0.0/sdk/keyvault/azure-security-keyvault-secrets/src/samples) | [Api Reference](https://azuresdkdocs.blob.core.windows.net/$web/java/azure-security-keyvault-secrets/4.0.0/index.html) | [ChangeLog](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-secrets_4.0.0/sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md) |
```
Comment (kaerm) [inline docs-ref-services/keyvault.md]: nit: I'd use README

File: docs-ref-services/keyvault.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ keywords: Azure, Java, SDK, API, keyvault, secure, keys, secrets, vault
 author: rloutlaw
 ms.author: routlaw
 manager: douge
-ms.date: 07/20/2017
+ms.date: 21/11/2019
```
Comment (kaerm) [inline docs-ref-services/keyvault.md]: 11/21/2019 :)

File: docs-ref-services/keyvault.md (hunk 1 above)
Comment (elraikhm) [reply docs-ref-services/keyvault.md]: done, please check if the link I added is correct

Comment (kaerm) [reply docs-ref-services/keyvault.md]: works for me - would just change the "Explore the Client APIs" to "Get started with Secrets"

File: docs-ref-services/keyvault.md (hunk 2 above)
Comment (elraikhm) [reply docs-ref-services/keyvault.md]: updated

File: docs-ref-services/keyvault.md (hunk 1 above)
Comment (elraikhm) [reply docs-ref-services/keyvault.md]: updated, please check the path

**Pull request merged**